lib/twilio.rb

```diff
@@ -0,0 +1,21 @@
+require 'twilio-ruby'
+
+class Twilio_sms
```

Comment:
Style/ClassAndModuleCamelCase: Use CamelCase for classes and modules. (https://github.com/bbatsov/ruby-style-guide#camelcase-classes)